app/components/Home/Yaroslav_Grushchak/Projects/Project_1/components/portfolio/portfolio.component.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+// import './portfolio.component.css';
+import './portfolio.component.styl';
+import PotrfolioCategoryButtons from '../portfolio-category-buttons/portfolio-category-buttons';
+
+const portfolio = [
+    {
+        id: 1,
+        class_name: 'thumb-1',
+        section: ['panoramas', 'macro']
+    },
+    {
+        id: 2,
+        class_name: 'thumb-2',
+        section: ['panoramas', 'journal']
+    },
+    {
+        id: 3,
+        class_name: 'thumb-3',
+        section: ['panoramas', 'macro']
+    },
+    {
+        id: 4,
+        class_name: 'thumb-4',
+        section: ['panoramas', 'journal']
+    },
+    {
+        id: 5,
+        class_name: 'thumb-5',
+        section: ['portraits', 'macro']
+    },
+    {
+        id: 6,
+        class_name: 'thumb-6',
+        section: ['portraits', 'journal']
+    },
+    {
+        id: 7,
+        class_name: 'thumb-7',
+        section: ['portraits', 'macro']
+    },
+    {
+        id: 8,
+        class_name: 'thumb-8',
+        section: ['portraits', 'journal']
+    }
+];
+
+class Portfolio extends React.Component
+{
+    constructor(){
+        super();
+        this.portfolio=portfolio;
+        this.state={section: 'all', filteredPortfolio: portfolio};
+    }
+    isEqual(item, section){
+        let equal = false;
+
+        item.section.forEach((s) => {
+            if(s == section){
+                equal = true;
+                return ;
+            }
+        });
+        return equal;
+    }
```

Comment:
Better to use array.some method
ES6 example:
isEqual = (item, section) => item.section.some(s => s === section)
Old JS example: 
isEqual = function (item, section) { return  item.section.some(function (s) { return s === section }) }

Reply:
Done